packages/now-cli/src/commands/alias/set.ts

```diff
@@ -133,62 +96,79 @@ export default async function set(
     return 1;
   }
 
-  // Find the targets to perform the alias
-  const targets = getTargetsForAlias(args, localConfig);
+  // For `now alias <argument>`
+  if (args.length === 1) {
+    const deployment = handleCertError(
+      output,
+      await getDeploymentForAlias(
+        client,
+        output,
+        args,
+        opts['--local-config'],
+        user,
+        contextName,
+        localConfig
+      )
+    );
 
-  if (targets instanceof ERRORS.NoAliasInConfig) {
-    output.error(`Couldn't find an alias in config`);
-    return 1;
-  }
+    if (deployment === 1) {
+      return deployment;
+    }
 
-  if (targets instanceof ERRORS.InvalidAliasInConfig) {
-    output.error(
-      `Wrong value for alias found in config. It must be a string or array of string.`
-    );
-    return 1;
-  }
+    if (deployment instanceof Error) {
+      output.error(deployment.message);
+      return 1;
+    }
 
-  if (rules) {
-    // If we have rules for path alias we assign them to the domain
-    for (const target of targets) {
-      output.log(
-        `Assigning path alias rules from ${humanizePath(
-          rulesPath
-        )} to ${target}`
+    if (!deployment) {
+      output.error(
+        `Couldn't find a deployment to alias. Please provide one as an argument.`
       );
-      const pathAlias = await upsertPathAlias(
+      return 1;
+    }
+
+    // Find the targets to perform the alias	  const [deploymentIdOrHost, aliasTarget] = args;
```

Comment:
Is the `const [deploymentIdOrHost, aliasTarget] = args;` supposed to be commented out?